winml/adapter/winml_adapter_model.cpp

```diff
@@ -120,7 +120,10 @@ OrtModel::OrtModel(std::unique_ptr<onnx::ModelProto> model_proto) : model_proto_
 // factory methods for creating an ort model from a path
 static OrtStatus* CreateModelProto(const char* path, std::unique_ptr<onnx::ModelProto>& out) {
   int file_descriptor;
-  auto wide_path = _winml::Strings::HStringFromUTF8(path);
+
+  auto path_str = std::string(path);
+  auto wide_path = onnxruntime::ToWideString(path_str);
```

Comment:
Does this test work?? On Chinese version Windows the path_str should be encoded in CP936. But ToWideString assumes it's utf-8. It's wrong.